package.json

```diff
@@ -26,8 +26,12 @@
   "devDependencies": {
     "@commitlint/cli": "16.3.0",
     "@commitlint/config-conventional": "16.2.4",
+    "@opengemeenten/iconset-react": "1.0.1",
+    "@opengemeenten/iconset-web-component": "1.0.0",
```

Comment:
Deze dependencies worden niet gebruikt, die kunnen denk ik weg:

- `@opengemeenten/iconset-react`
- `@opengemeenten/iconset-web-component`
- `@utrecht/icon`

Ik denk dat er dan effectief geen wijzigingen zijn voor zowel `package-lock.json` en `package.json`, dan kun je dit doen:

```shell
git checkout origin/main package.json
git checkout origin/main package-lock.json
```

Reply:
Oh, ik zie dat `@utrecht/icon` wel gebruikt wordt, maar met name in de storybook package. Dan kun je die dependencie het beste alleen daar installeren, met:

```shell
npm install --workspace packages/storybook/ --save-dev @utrecht/icon
```

of door `"@utrecht/icon": "file:../../components/icon",` toe te voegen aan die `package.json`